infrastructure/main.tf

```diff
@@ -0,0 +1,33 @@
+resource "random_string" "password" {
+  length = 32
+  special = true
+}
+
+module "api" {
+  source   = "git@github.com/hmcts/terraform-module-webapp.git"
+  product  = "${var.product}-api"
+  location = "${var.location_api}"
+  env      = "${var.env}"
+  asename  = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+
+  app_settings = {
+    DRAFT_STORE_DB_HOST     = "${module.db.host_name}"
+    DRAFT_STORE_DB_PORT     = "${module.db.postgresql_listen_port}"
+    DRAFT_STORE_DB_PASSWORD = "${random_string.password.result}"
+
+    IDAM_URL                = "http://idam-${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
+    S2S_URL                 = "http://idam-s2s-${var.env}.service.${data.terraform_remote_state.core_apps_compute.ase_name[0]}.internal"
```

Comment:
for the record, none of those exist yet

Reply:
that's fine - the infra will build ok and url will be exposed ok to the app - it obvs just won't work when you try to call it